src/JsonApiDotNetCore.Annotations/Resources/Annotations/AttrAttribute.cs

```diff
@@ -21,7 +21,7 @@ public sealed class AttrAttribute : ResourceFieldAttribute
     /// public class Author : Identifiable<long>
     /// {
     ///     [Attr(Capabilities = AttrCapabilities.AllowFilter | AttrCapabilities.AllowSort)]
-    ///     public string Name { get; set; } = null!;
+    ///     public required string Name { get; set; }
```

Comment:
```suggestion
    ///     public string? Name { get; set; }
```
To simplify the example, nullability/requiredness is not important here.